src/DependencyInjection/SonataCoreExtension.php

```diff
@@ -196,11 +196,9 @@ public function configureSerializerFormats($config)
     protected function deprecateSlugify(ContainerBuilder $container)
     {
         $definition = $container->getDefinition('sonata.core.slugify.cocur');
-        if (method_exists($definition, 'setDeprecated')) {
-            $definition->setDeprecated(true);
+        $definition->setDeprecated(true);
```

Comment:
Can be simplified further: `$container->getDefinition('sonata.core.slugify.cocur')->setDeprecated(true);`, same below